Source/Analytics/Web.Frontend/src/components/HeaderPanel.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Heading, UnorderedList, ListItem } from "evergreen-ui";
+
+export class HeaderPanel extends React.Component {
```

Comment:
I agree! I renamed it to PresenceIndicator, that fits with its current use, but it can be used for many other things, so I'm open to name suggestions:)